Catrobat/Paintroid
# [GSoC18] Convert Pocket Paint to android library module

* Change project type to library
* Add sample application (app module)
* Speed up DrawingSurface by setting the proper pixel format
* Remove unused features
	* Remove MultilingualActivity and its resources
	* Move language features to tests
	* Fix language related tests
	* Remove TOS dialog
	* Remove About dialog
	* Add version notice on bottom of navigation sidebar
* Fix resource property
	* `resources` defines _java_ resources, it should be `res` which defines android resources.
* Clean up lint warnings
